.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "site-shared"]
   path = site-shared
   url = https://github.com/dart-lang/site-shared.git
-  branch = master
+  branch = chalin-scss-bs-config-1018
```

Comment:
Using a branch so as to avoid forcing changes on other sites that are using master.